integration/travis/run_integration.sh

```diff
@@ -107,6 +112,24 @@ case "$COOK_POOLS" in
     exit 1
 esac
 
+case "$JOB_LAUNCH_RATE_LIMIT" in
+    on)
+      # Note: Carefully chosen for XXX unit test.
```

Comment:
> XXX

?

Reply:
Was left to fill-in-integration-test-name. Named filled in.